### PR TITLE
Add support for statistics and filegen

### PR DESCRIPTION
**Pull request by oliviazhang0809**

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -4,6 +4,7 @@
   $config_template   = $ntp::params::config_template,
   $disable_monitor   = $ntp::params::disable_monitor,
   $driftfile         = $ntp::params::driftfile,
+  $statistics         = $ntp::params::statistics,
```
Comment (igalic) [inline manifests/init.pp]: is there a reason you're not aligning these?

File: manifests/init.pp [hunk 2]
```diff
@@ -16,6 +17,7 @@
   $panic             = $ntp::params::panic,
   $preferred_servers = $ntp::params::preferred_servers,
   $restrict          = $ntp::params::restrict,
+  $filegen          = $ntp::params::filegen,
```
Comment (igalic) [inline manifests/init.pp]: is there a reason you're not aligning these?

File: manifests/params.pp [hunk 3]
```diff
@@ -25,6 +25,7 @@
   $default_config       = '/etc/ntp.conf'
   $default_keys_file    = '/etc/ntp/keys'
   $default_driftfile    = '/var/lib/ntp/drift'
+  $default_statistics = 'loopstats peerstats clockstats'
```
Comment (igalic) [inline manifests/params.pp]: is there a reason you're not aligning these?

**Comment by igalic** [comment]
this is missing an addition to README and spec tests
***
you might also want to consider using the [github flow](https://guides.github.com/introduction/flow/index.html) for your next PR, this will make things like updating ([rebasing](http://git-scm.com/book/en/Git-Branching-Rebasing)) easier

**Comment by mhaskel** [comment]
@oliviazhang0809 ping?

**Comment by tphoney** [comment]
Thanks for the work you have put in to this pull request. However due to inactivity and failing unit tests, no documentation, needs rebased, no tests added.
This pull request is being closed. If you want to re-submit this pr with the necessary changes, we will look at the new pull request.

**Pull request closed without merging**